server/api/costumes.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-ignore
+import express from "express";
+
+const { getAllCostumes, getPool } = require("../db");
+
+const pool = getPool();
```

Comment:
Oh this is going to have some serious performance issues in the long-run, because it's creating a new pool for every single request

I filed an issue in the backlog for refactoring this later: #52